### ISSUE TITLE
Add user to mailing list automatically once membership is approved

### ISSUE THREAD

**Issue by iambibhas** [pythonindia/pssi.org.in#40]
Need to check if it's possible with mailman to do it automatically.

**Comment by vnbang2003** [comment]
I didn't find any api support for mailman.
For existing member we are adding in manually. 

I think we can close as won't fix.

**Comment by vnbang2003** [comment]
closing it as won't fix.
